docs/fsharp/language-reference/import-declarations-the-open-keyword.md

```diff
@@ -61,11 +61,11 @@ Some namespaces are so frequently used in F# code that they are opened implicitl
 |`Microsoft.FSharp.Text`|Contains functions for formatted IO, such as the `printf` function.|
 
 ## AutoOpen Attribute
-You can apply the `AutoOpen` attribute to an assembly if you want to automatically open a namespace or module when the assembly is referenced. You can also apply the `AutoOpen` attribute to a module to automatically open that module when the parent module or namespace is opened. For more information, see [Core.AutoOpenAttribute Class](https://stage.docs.microsoft.com/en-us/dotnet/articles/fsharp/language-reference/core.autoopenattribute-class).
+You can apply the `AutoOpen` attribute to an assembly if you want to automatically open a namespace or module when the assembly is referenced. You can also apply the `AutoOpen` attribute to a module to automatically open that module when the parent module or namespace is opened. For more information, see [Core.AutoOpenAttribute Class](https://msdn.microsoft.com/visualfsharpdocs/conceptual/core.autoopenattribute-class-%5bfsharp%5d).
```

Comment:
Yep, this topic hasn't been moved to docs yet and I talked to @cartermp offline to confirm the change before I submitted this PR.